peps/pep-0768.rst

```diff
@@ -2,11 +2,12 @@ PEP: 768
 Title: Safe external debugger interface for CPython
 Author: Pablo Galindo Salgado <pablogsal@python.org>, Matt Wozniski <godlygeek@gmail.com>, Ivona Stojanovic <stojanovic.i@hotmail.com>
 Discussions-To: https://discuss.python.org/t/pep-768-safe-external-debugger-interface-for-cpython/73969
-Status: Draft
+Status: Final
```

Comment:
Accepted now, and Final when https://github.com/python/cpython/issues/131591 is done :)

```suggestion
Status: Accepted
```

We'll also add a `canonical-docs` link when marking Final.

https://github.com/python/peps/blob/main/.github/PULL_REQUEST_TEMPLATE/Mark%20a%20PEP%20Final.md